### ISSUE TITLE
add option to output matrix column/row names as needed for different MSA tools

### ISSUE THREAD

**Issue by carjed** [carjed/helmsman#2]

**Comment by carjed** [comment]
Rather than doing this directly in Helmsman, direct users to a separate R package with functions for formatting mutation spectra matrices between tools.

When Helmsman is passed some parameter, e.g., `--tool SomaticSignatures`, output instructions for loading the matrix in R and coercing to the format required by the specified tool.

```r
library("SomaticSignatures")
library("devtools")
install_github("carjed/musigtools")
library("musigtools")

mu_counts <- read.table("/path/to/projectdir/subtype_count_matrix.txt", header=T, stringsAsFactors=F)
msm <- format_counts(mu_counts, "SomaticSignatures")
```

**Comment by carjed** [comment]
fixed in 963cf87
